molecule/signature/playbook.yml

```diff
@@ -33,10 +33,17 @@
           - openssh-server
         state: present
 
-    # don't know why this is needed...
+    - name: Ensure sshd service is started
+      service:
+        name: ssh
+        state: started
+      when: ansible_distribution == 'Ubuntu' and ansible_distribution_version == '14.04'
+
     - name: Ensure sshd service is started
       service:
         name: sshd
         state: started
+      when: not (ansible_distribution == 'Ubuntu' and ansible_distribution_version == '14.04')
+
```

Comment:
...moreover this duplication is also present across scenarios. Factorisation look beneficial. I'm ok postponing this into a separate issue and merging this version first.

Reply:
Indeed, and it's already done, will be in the next PR, you can take a look in macno/tests branch

Reply:
great!